mozilla_aws_cli/static/index.js

```diff
@@ -107,6 +107,9 @@ const pollState = setInterval(async () => {
         // show the roles
         const roles = await response.json();
         showRoles(roles);
+    } else if (remoteState.state === "restart_auth") {
+        setMessage("Redirecting to identity provider...");
```

Comment:
You'll probably want to `sleep()` here, if you want this message to be seen.

Reply:
I don't actually care if the user sees it, just figured I'd put it up in case the user had a slow connection or a network issue so they can see what's going on.